examples/js/fortunewhale.js

```diff
@@ -0,0 +1,168 @@
+const fw = require("fission-workflows");
+
+// Single-task workflow
+module.exports = function () {
+    return fw.task({
+        run: "fortune"
+    })
+};
+
+// Simple dependencies with tasks in the requires
+module.exports = function () {
+    const generateFortune = fw.Task({
+        id: "generateFortune",
+        run: "fortune"
+    });
+
+    const whaleSays = fw.task({
+        run: "whalesay",
+        inputs: {
+            "body": "{ output('generateFortune') }"
+        },
+        requires: [
+            generateFortune
+        ]
+    });
+
+    return fw.Workflow({
+        output: generateFortune,
+        tasks: [
+            generateFortune,
+            whaleSays
+        ]
+    })
+};
+
+// Simple dependencies with task IDs in the requires
+module.exports = function () {
+    const generateFortune = fw.Task({
+        id: "generateFortune",
+        run: "fortune"
+    });
+
+    const whaleSays = fw.task({
+        run: "whalesay",
+        inputs: {
+            "body": "{ output('generateFortune') }"
+        },
+        requires: [
+            generateFortune.id
+        ]
+    });
+
+    return fw.Workflow({
+        output: generateFortune,
+        tasks: [
+            generateFortune,
+            whaleSays
+        ]
+    })
+};
+
+// Referencing a task in an expression without an ID.
+module.exports = function () {
+    const generateFortune = fw.Task({
+        run: "fortune"
+    });
+
+    const whaleSays = fw.task({
+        run: "whalesay",
+        inputs: {
+            "body": `{ output('${generateFortune.id}') }`,  // or:
+            "body2": output(generateFortune.id),            // or:
+            "body3": output(generateFortune),
```

Comment:
why not simply `generateFortune.output`

Reply:
See https://github.com/fission/fission-workflows/pull/200/files#r215565319